setup.cfg

```diff
@@ -36,6 +36,7 @@ setuptools.finalize_distribution_options =
 
 [options.extras_require]
 test =
+    wheel
```

Comment:
Why?

Reply:
The test C extension package failed otherwise for some reason.

Reply:
curious